### PR TITLE
cli: Do not open session in reading `object` commands

### LINKED ISSUE #1865: It makes no sense to open sessions for reading objects
In current implementation NeoFS CLI opens session with the remote node in next `object` commands:

- `get`
- `head`
- `search`
- `range`
- `hash`

The session is unconditionally issued by the command using NeoFS API `SessionService` service. It doesn't really makes sense for these commands, since session won't have any useful effect like in `put` / `delete` commands.

I propose to not open session if corresponding flag with encoded token is provided.

### PR DESCRIPTION
**Pull request by cthulhu-rider**
* closes #1865

### CODE REVIEW

**Review by fyrchik — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Changelog for NeoFS Node
 - Flush write-cache when moving shard to DEGRADED mode (#1825)
 - Make `morph.cache_ttl` default value equal to morph block time (#1846)
 - Policer marks nodes under maintenance as OK without requests (#1680)
+- Reading `object` commands of NeoFS CLI don't open remote sessions (#1865)
```
Comment (carpawell) [inline CHANGELOG.md:27]:
```suggestion
- Reading `object` commands of NeoFS CLI doesn't open remote sessions (#1865) 
```

File: cmd/neofs-cli/modules/object/util.go [hunk 2]
```diff
@@ -81,3 +86,89 @@ func readOID(cmd *cobra.Command, id *oid.ID) {
...
+}
+
+// decodes object session from JSON file from "session" command flag if it is provided,
+// and writes resulting session into the provided sessionPrm. Checks:
+//
+//	if session verb corresponds to given sessionPrm according to its type
+//	relation to the given container
+//	relation to the given object if non-nil
+//	relation to the given private key used withing the command
+//	session signature
```
Comment (carpawell) [inline cmd/neofs-cli/modules/object/util.go:113]: isn't it hard to read?

**Review by carpawell — APPROVED** [review]

File: cmd/neofs-cli/modules/object/util.go (hunk 2 above)
Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/object/util.go:113]: How can I improve readability?

File: CHANGELOG.md (hunk 1 above)
Comment (cthulhu-rider) [reply CHANGELOG.md:27]: I wanted `commands` to be subject, I'll try to fix grammar

Comment (carpawell) [reply CHANGELOG.md:27]: oh that's my fault, i read that as "reading ... don't", dont remember why

you were correct

File: cmd/neofs-cli/modules/object/util.go (hunk 2 above)
Comment (carpawell) [reply cmd/neofs-cli/modules/object/util.go:113]: we usually use `-` in such cases. it is hard for me to understand where the sentence ends

lets discuss

Comment (cthulhu-rider) [reply cmd/neofs-cli/modules/object/util.go:113]: Agree with message excretion, added dashes.

**Pull request merged**